packages/core/src/index.ts

```diff
@@ -457,7 +456,7 @@ function prepareSources(target: Computed | Effect) {
 
 function cleanupSources(target: Computed | Effect) {
 	let node = target._sources;
-	let head = undefined;
+	let head: Node | undefined = undefined;
```

Comment:
```suggestion
	let head: Node;
```

This should be sufficient for TS